.eslintrc.js

```diff
@@ -49,9 +50,20 @@ module.exports = {
         "@typescript-eslint/ban-ts-comment": "off",
         "@typescript-eslint/no-empty-interface": "off",
         "@typescript-eslint/no-explicit-any": "off",
-        "@typescript-eslint/no-var-requires": "off",
         "@typescript-eslint/semi": "off",
+        "@typescript-eslint/no-var-requires": "warn",
```

Comment:
Q: is there a standard we're following?
Styling is always relative just curious what are the baselines and preferences

Reply:
I have left what was added before in this file and just added a few more rules, like no-trailing-spaces and prefer-const which are generally recommended. Above in the extended property you can see we support what is recommended by defaul and we turn off some of those rules for our projects, like semicolons, i am open to discussion on what should be on and off